dbms/src/TestUtils/mockExecutor.h

```diff
@@ -52,9 +53,10 @@ class DAGRequestBuilder
         return executor_index;
     }
 
-    explicit DAGRequestBuilder(size_t & index)
+    explicit DAGRequestBuilder(size_t & index, Int32 collator = TiDB::ITiDBCollator::UTF8MB4_BIN)
         : executor_index(index)
     {
+        properties.collator = -abs(collator);
```

Comment:
Why do we use `-abs(collator)`?

Reply:
> Why do we use `-abs(collator)` instead of just `collator`?

collator passed by tidb is negative, so we need to guarantee the collator < 0.  See https://github.com/pingcap/tiflash/blob/master/dbms/src/Flash/Coprocessor/DAGUtils.cpp#L1357